### DISCUSSION TITLE
Distributed Multi Agent with workflows example.

### DISCUSSION THREAD

**Discussion by ravisamavedula** [run-llama/llama_deploy#261] [General]
Hi,

Llama_Deploy is going in right direction for scaling agents with their architecture, could you publish a Distributed multi-agent workflow/Sample code, it will ease in using and users can give feedback.

**Comment by altonalexander** [comment]
I'm also very interested in multi agent and the latest release v0.2.4+ and the ability to maintain a session between my agents and the frontend user.

see the [fullstack example](https://github.com/run-llama/llama_deploy/tree/287abb9509cb652fa6c54fcf236d6a90cd1ef4cc/examples/python_fullstack) including: session = client.get_or_create_session("fake_session_id")
